src/morgan/morgan.middleware.ts

```diff
@@ -2,27 +2,61 @@ import * as morgan from "morgan";
 import { MORGAN_FORMAT_STRING } from "./morgan.constants";
 import { NestjsWinstonLoggerService } from "../nestjs-winston-logger.service";
 
+interface MorganOptions {
+  skip?: (request: Request, response: Response) => boolean;
+  format?: string;
+}
+
+const normalizeOptions = (
+  options: string | MorganOptions,
```

Comment:
I believe it's better to set the type of options to MorganOptions so that the code would be cleaner.
But there is a draw back that this update would become a breaking change.
What do you think @Marcotsept ?

Reply:
yes was trying to avoid the breaking change but i'm fine with it once you guys make the decision